yarn-project/bb-prover/src/prover/bb_prover.ts

```diff
@@ -380,22 +379,21 @@ export class BBNativeRollupProver implements ServerCircuitProver {
       }
 
       // Ensure our vk cache is up to date
-      await this.updateVerificationKeyAfterProof(provingResult.vkPath!, circuitType);
+      const vkData = await this.updateVerificationKeyAfterProof(provingResult.vkPath!, circuitType);
 
       // Read the proof and then cleanup up our temporary directory
       const proof = await this.readProofAsFields<PROOF_LENGTH>(provingResult.proofPath!, circuitType);
 
       logger.info(
         `Generated proof for ${circuitType} in ${provingResult.duration} ms, size: ${proof.proof.length} fields`,
-      );
-
-      emitCircuitProvingStats(
-        circuitTypeToCircuitName(circuitType),
-        provingResult.duration,
-        witnessMap.size * Fr.SIZE_IN_BYTES,
-        outputWitness.size * Fr.SIZE_IN_BYTES,
-        proof.binaryProof.buffer.length,
-        logger,
+        {
+          circuitName: mapProtocolArtifactNameToCircuitName(circuitType),
+          circuitSize: vkData.circuitSize,
+          duration: provingResult.duration,
+          inputSize: witnessMap.size * Fr.SIZE_IN_BYTES,
+          proofSize: proof.binaryProof.buffer.length,
```

Comment:
Can you have 2 columns. One of which is the actual proof size, and the other the number of public inputs (available on `VerificationKeyData`)?